src/gsy_e/models/myco_matcher/myco_external_matcher.py

```diff
@@ -84,19 +84,25 @@ def publish_orders(self, message):
             for market in area_data["markets"]:
                 self.area_markets_mapping.update(
                     {f"{area_uuid}-{market.time_slot_str}": market})
-                market_offers_bids_list_mapping[area_uuid] = self._get_orders(market, filters)
+                if area_uuid not in market_orders_list_mapping:
+                    market_orders_list_mapping[area_uuid] = {}
```

Comment:
Do we need to add the empty area to the mapping? Couldn't we just leave it out?

Reply:
In the next line we're guaranteed to add orders to this empty area mapping.
I added the key here in order to aggregate orders of area later on (An area can have orders in different time slots)